### PR TITLE
Update allow-members-to-send-as-or-send-on-behalf-of-group.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a related article which talks about how to send as DL.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

Note that the relative link is not working and I had to use the absolute link. Can you check if I'm doing anything wrong here?

**Review by JohanFreelancer9 — APPROVED** [review]

File: microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md [hunk 1]
```diff
@@ -81,4 +81,6 @@ This section explains how to allow users to send email on behalf of a group in t
 
 [Add-RecipientPermission](/powershell/module/exchange/add-recipientpermission)
 
-[Set-UnifiedGroup](/powershell/module/exchange/set-unifiedgroup)
\ No newline at end of file
+[Set-UnifiedGroup](/powershell/module/exchange/set-unifiedgroup)
+
+[How to Send email as a distribution list](https://docs.microsoft.com/microsoft-365/admin/manage/send-email-as-distribution-list)
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md]:
```suggestion
[How to Send email as a distribution list](/admin/manage/send-email-as-distribution-list)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@msbemba - Relative links might not work in the preview version for the PR (a link which should be generated after you add the file extension to the file name), but they'll work when the article is merged to master. You can test this by changing **branch=pr-en-us-7707** in the preview URL to **branch=master**.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks

**Comment by MikePlumleyMSFT** [comment]
@msbemba there is already a link to [Send email from or on behalf of an Office 365 group](https://support.microsoft.com/office/0f4964af-aec6-484b-a65c-0434df8cdb6b) in the green Tip near the top.

**Pull request closed without merging**